Tests/IntegrationTests/SwiftKafkaTests.swift

```diff
@@ -201,7 +201,7 @@ final class SwiftKafkaTests: XCTestCase {
         var messageIDs = Set<UInt>()
 
         for message in messages {
-            messageIDs.insert(try await producer.sendAsync(message))
+            try messageIDs.insert(await producer.sendAsync(message))
```

Comment:
We usually prefer to disable `hoistTry` (and `hoistAwait`) to avoid moving `try`s and `await`s to the beginning of the line. It's generally not useful to lose clarity of what is throwing/async, and sometimes actually generates compiler errors because you can end up with malformed syntax.

You can disable both options by adding the following to `.swiftformat`:
```
--disable hoistTry
--disable hoistAwait
```

You can read more about the options [here](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#hoistTry) and [here](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#hoistAwait).